contracts/allowlist/OperatorAllowlistUpgradeable.sol

```diff
@@ -0,0 +1,189 @@
+// Copyright Immutable Pty Ltd 2018 - 2023
+// SPDX-License-Identifier: Apache 2.0
+pragma solidity 0.8.19;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+
+// Introspection
+import {ERC165} from "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+
+// Interfaces
+import {IOperatorAllowlist} from "./IOperatorAllowlist.sol";
+
+// Interface to retrieve the implemention stored inside the Proxy contract
+interface IProxy {
```

Comment:
This should be a separate file. Doing this will allow the same interface file to be used by all usages, rather than the code being copy and pasted.

The file name should be IWalletProxy. The description should be:

Interface for Passport Wallet's proxy contract.